plugins/catalog-backend-module-openapi/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-openapi",
+  "description": "A Backstage catalog backend module that helps with OpenAPI specifications",
+  "version": "0.1.0",
```

Comment:
Make this `0.0.0`, the changeset will make sure the first version released will be 0.1.0

Reply:
Updated now.